test/hotspot/gtest/runtime/test_synchronizer.cpp

```diff
@@ -66,3 +71,34 @@ TEST_VM(SynchronizerTest, sanity) {
             << "sharing.";
   }
 }
+
+TEST_VM(SynchronizerTest, monitorListStats) {
+  JavaThread* THREAD = JavaThread::current();
+  ThreadInVMfromNative invm(THREAD);
+  ResourceMark rm(THREAD);
+
+  // Something to reference in OM. It makes no difference which oop it is,
+  // as long as it is correct.
```

Comment:
Can’t we still safepoint on `workers.doit()` when calling `VMThreadBlocker::start()`? Maybe we could use a Handle to be safe. Sidenote: This is pre-existent but isn’t `TestThreadGroup::doit()` missing a call to `_blocker->ready()` before starting the worker threads?